znflow/node.py

```diff
@@ -51,14 +51,13 @@ def __new__(cls, *args, **kwargs):
         return instance
 
     def __getattribute__(self, item):
-        value = super().__getattribute__(item)
         if get_graph() is not None:
             if item not in type(self)._protected_ and not item.startswith("_"):
                 if self._in_construction:
-                    return value
+                    return super().__getattribute__(item)
```

Comment:
Now the issue is, that one can do `node.abcdefg` and the `AttributeError` will only occur an runtime.

Reply:
resolved in [7aee678](https://github.com/zincware/ZnFlow/pull/42/commits/7aee678dd88f4633c107e856909392063165d29e)